src/Caller/Masa.Utils.Caller.Core/IRequestIdGenerator.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Utils.Caller.Core;
+
+public interface IRequestIdGenerator
```

Comment:
I think you can use IdGenerator.

Reply:
Later migration to BuildingBlock will adjust